### ISSUE TITLE
TreeMap back button issue

### ISSUE THREAD

**Issue by mrerduran** [highcharts/highcharts#16353]
I have a treemap with settings which are below but when I drill in a node, I cannot access the back button. I cannot focus on it with the keyboard. I could not find accessibility settings on the forum and documents. I can miss some points but does highcharts has accessibility to the back button? How can I access that button with the keyboard? Is there any parameter for it?

`   series: [{
                type: 'treemap',
                layoutAlgorithm: 'squarified',
                allowDrillToNode: true,
                animationLimit: 1000,
                dataLabels: {
                    enabled: false
                },
                levelIsConstant: false,
                levels: [{
                    level: 1,
                    dataLabels: {
                        enabled: true
                    },
                    borderWidth: 0
                }],
                data: points
            }],`

**Comment by KacperMadej** [comment]
Hi @erduranogulcan 

Please add a live demo.

**Comment by erduranogulcan** [comment]
I use a demo large treemap chart as template. So, it is not different from my chart. You can see the demo below, I cannot click and hover the back button with the keyboard. @KacperMadej 

https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/demo/treemap-large-dataset

**Comment by erduranogulcan** [comment]
![image](https://user-images.githubusercontent.com/33247817/134298170-b4880547-1a02-4919-92fa-1a6a3c17172a.png)

**Comment by raf18seb** [comment]
@erduranogulcan thanks for providing more info. Indeed the button is not accessible in the treemap chart. Works fine with a `drilldown.js` module (but treemap doesn't use a standard `drilldown.js`).

Seems like a feature request. @oysteinmoseng can you confirm?

**Comment by oysteinmoseng** [comment]
I think we'll mark this as a bug, we obviously should add the same functionality to this button as the button in `drilldown.js`.
